src/App.js

```diff
@@ -1,10 +1,32 @@
+import React from 'react';
 import Calculator from './components/Calculator';
 import './App.css';
+import Quotes from './components/quotes';
 
 function App() {
+  const [quote, setQuote] = React.useState(['loading...']);
```

Comment:
- [ OPTIONAL  ] I think it would be great if you add a state for the error and state for the loading and another state for the quotes istfelf as mentioed in the requirments 🚀⭐🔥  I could see that you added loading state tp the qoutes but it would be better if you add as a seprate state 🙏🏻

![image](https://github.com/rachaouldbabaali/math-magicians/assets/58553711/48937957-fbfa-4fbb-a49e-bed75320d20c)